crates/spfs-cli/cmd-monitor/src/cmd_monitor.rs

```diff
@@ -142,12 +144,23 @@ impl CmdMonitor {
     }
 
     pub async fn run_async(&mut self, config: &spfs::Config) -> Result<i32> {
+        #[cfg(unix)]
         let mut interrupt = signal(SignalKind::interrupt())
             .map_err(|err| Error::process_spawn_error("signal()", err, None))?;
+        #[cfg(windows)]
+        let mut interrupt =
+            ctrl_c().map_err(|err| Error::process_spawn_error("ctrl_c()", err, None))?;
+        #[cfg(unix)]
         let mut quit = signal(SignalKind::quit())
             .map_err(|err| Error::process_spawn_error("signal()", err, None))?;
+        #[cfg(windows)]
+        let mut quit = ctrl_c().map_err(|err| Error::process_spawn_error("ctrl_c()", err, None))?;
+        #[cfg(unix)]
         let mut terminate = signal(SignalKind::terminate())
             .map_err(|err| Error::process_spawn_error("signal()", err, None))?;
+        #[cfg(windows)]
+        let mut terminate =
+            ctrl_c().map_err(|err| Error::process_spawn_error("ctrl_c()", err, None))?;
```

Comment:
To avoid all this `#[cfg]` maybe we can create a single future to wrap all the signals - 

specifically, define a trait with a function that builds a future for all signals in one that can be implemented differently on this `CmdMonitor` type for each os. Let me know if my description is not clear